File: 5400-s-master/src/main/java/pl/twojadruzyna/model/Player.java
package pl.twojadruzyna.model;

import lombok.Data;
import pl.twojadruzyna.model.enums.Card;
import pl.twojadruzyna.model.enums.Gender;
import pl.twojadruzyna.model.enums.Position;

import javax.persistence.*;

@Entity
@Data
public class Player {
    @Id
    @GeneratedValue
    private Long id;
    private String firstName;
    private String secondName;
    @ManyToOne(cascade= CascadeType.ALL, fetch= FetchType.EAGER)
    private Team team;
    private Position position;
    private Gender gender;
    private Integer goals;
    private Integer shots;
    private Integer faults;
    private Card card;
    private boolean captain;
    private boolean injured;
    private int age;
}

File: twojadruzynfront-master/src/app/common/model/player.ts
import { Team } from "./team";

export class Player {
    id: number;
    firstName: string;
    secondName: string;
    team: Team;
    position: string;
    gender: string;
    goals: number;
    shots: number;
    faults: number;
    card: string;
    captain: boolean;
    injured: boolean;
    age: number
}
